### ISSUE TITLE
Rule idea: Report empty constructor/destructor blocks

### ISSUE THREAD

**Issue by movedoa** [Riverside-Software/sonar-openedge#213]
From time to time i find these empty constructor/destructor blocks.
I guess they come from refactoring or lack of understanding.

A rule to report these empty blocks would be nice just to keep the code clean.
I can't think of a situation where deleting empty constructor/destructor blocks would lead to problems.

**Comment by gquerret** [comment]
EmptyBlock rule has been added a few days ago: http://sonar.riverside-software.fr/coding_rules#rule_key=rssw-oe-main%3Aeu.rssw.antlr.proparse.checks.EmptyBlock
Methods are currently skipped as there were false positives, but that will be fixed soon

**Comment by gquerret** [comment]
Fixed in master.
Now only reports code blocks where there are no statements **and** no comments.

**Comment by maelstraggiotti** [comment]
EmptyBlock rule reports an empty constructor for this case, where it's required : 
```
CLASS Example:

    CONSTRUCTOR PUBLIC Example():
    END CONSTRUCTOR.

    CONSTRUCTOR PUBLIC Example(
        INPUT picValue AS CHARACTER
        ):

        THIS-OBJECT:Value = picValue.
    END CONSTRUCTOR.

    DEFINE PUBLIC PROPERTY Value AS CHARACTER NO-UNDO
    GET.
    PRIVATE SET.

END CLASS.
```
It's a common case, can I avoid suppressing it explicitly ?

**Comment by gquerret** [comment]
You can include a comment to suppress the issue. My usual comment when I don't want to explain why a method is empty is `// No-op`, but you may want to explain in the comment why there's no implementation for the method
